src/components/GoodList/GoodList.jsx

```diff
@@ -0,0 +1,9 @@
+import { GoodCard } from '../GoodCard';
+
+export const GoodList = ({ goods }) => (
+  <>
+    {goods.map(good => (
+      <GoodCard good={good} key={good} />
+    ))}
+  </>
+);
```

Comment:
This component should be converted to TypeScript. Define the prop types using TypeScript interfaces or types to ensure type safety.